tests/foreman/ui/test_syncplan.py

```diff
@@ -81,19 +81,22 @@ def module_org():
 
 
 @tier2
-def test_positive_end_to_end(session):
+def test_positive_end_to_end(session, module_org):
     """Perform end to end scenario for sync plan component
 
     :id: 39c140a6-ca65-4b6a-a640-4a023a2f0f12
 
     :expectedresults: All CRUD actions for component finished successfully
 
     :CaseLevel: Integration
+
+    :BZ: 1693795
     """
     plan_name = gen_string('alpha')
     description = gen_string('alpha')
     new_description = gen_string('alpha')
     with session:
+        session.contenthost.read_all('current_user')
```

Comment:
I see you are generous with comments below, so maybe add a comment here as well:
```python
        # workaround: force session.browser to point to browser object on next line
        session.contenthost.read_all('current_user')
```

Of course my ACK still stands and this is not blocking

Reply:
OK, done that